tests/xmltester/tests/general/TestPreparedPolygonPredicate.xml

```diff
@@ -18,7 +18,21 @@
 </case>
 
 <case>
-  <desc>A/P - point equal to start point of polygon
+  <desc>A/P - point in polygon interior
```

Comment:
Good improvements to the test coverage.  I will port these over for sure, in preparation for other improvements.